src/const.js

```diff
@@ -0,0 +1,7 @@
+export const EVENTS_LIST_TIME_FORMAT = 'HH:mm';
+export const EVENTS_LIST_DATE_FORMAT = 'MMM DD';
+export const EVENT_EDIT_DATE_FORMAT = 'DD/MM/YY HH:mm';
```

Comment:
Объединить в общий объект

```suggestion
export const EVENTS_LIST_TIME_FORMAT = 'HH:mm';
export const EVENTS_LIST_DATE_FORMAT = 'MMM DD';
export const EVENT_EDIT_DATE_FORMAT = 'DD/MM/YY HH:mm';
```